### DISCUSSION TITLE
[Feature Request] Disable Thumbnail On Embed

### DISCUSSION THREAD

**Discussion by pxwee5** [vimeo/player.js#675] [Ideas]
### Expected Behavior
Enable `thumbnail: false` embed option to disable showing of thumbnail on player. 

`true`: The player should show the thumbnail from the control panel settings.
`false`: The player should show the first frame and a play button.
![image](https://user-images.githubusercontent.com/14123790/67354466-a81cff80-f5b1-11e9-85b2-196f0c9edea4.png)

### Actual Behavior
No way to disable thumbnail with current setup.
![image](https://user-images.githubusercontent.com/14123790/67354449-99364d00-f5b1-11e9-8d37-f43d344e0f38.png)

### Steps to Reproduce
Example: https://player.vimeo.com/api/demo

**Comment by jelias** [comment]
@fisherinnovation I saw that this issue was marked as completed... was this actually implemented? Is there documentations for how to control this setting in the API? would it just be a query param `?thumbnail=0`?
